frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -55,3 +116,38 @@ const getAttachmentsMap = (
 
   return attachmentsMap
 }
+const filterEmailResponsesWithAutoreply = (
```

Comment:
to be clear, we are changing the submission API call here right? because currently we are sending all email and mobile fields

Reply:
Yes, there is no need to send anything other than email responses without autoreply.

Reply:
7eaac53